lib/addMethod/globalize/string.js

```diff
@@ -2,28 +2,28 @@
     Given the keys for both local and global field, return the substituted
     string with global prepending the local string
 */
-var _          = require('lodash');
+const _ = require('lodash');
 
-var substitute = require('../substitute');
-var localOnly = require('./localOnly');
+const substitute = require('../substitute');
+const localOnly = require('./localOnly');
 
 module.exports = function (config, params, key, globalKey) {
 
-    globalKey = globalKey || key;
+	globalKey = globalKey || key;
 
-    if (localOnly(config, globalKey)) {
-        return substitute(config[key], params) || '';
-    }
+	if (localOnly(config, globalKey)) {
+		return substitute(config[key], params) || '';
+	}
 
-    //Ordering is significant - if globals true, global should run first
-    var subbedGlobalString = substitute(this._globalOptions[globalKey], params) || '',
-        subbedMethodString = substitute(config[key], params) || '';
+	//Ordering is significant - if globals true, global should run first
+	const subbedGlobalString = substitute(this._globalOptions[globalKey], params) || '',
+		subbedMethodString = substitute(config[key], params) || '';
```

Comment:
this seems to be the same thing as on line 15

Reply:
The code is written like that to preserve order of processing. If global is to be done, it should be processed first, before method.

Given modification by reference is still possible in areas outside of before/beforeRequest - I think it's safer keeping it like this until v2 where it can be corrected and be classified as a potential breaking change.